README.md

```diff
@@ -11,79 +12,88 @@ Country-specific resources are marked with a flag and country icons, like :trian
+- PowerToFly - diversity-focused **virtual career fair**: [link](https://powertofly.com/events/virtual-job-fair/) :rainbow:
+- TechTO Talent - livestreams and networking in the **Canadian startup** community: [link](https://www.techto.org/livestreams) :triangular_flag_on_post::canada:
+- Your local **meetups**: [link](https://www.meetup.com/)
+- **DevPost** The home for hackathons: [link](https://devpost.com/)
 
+## :family: Communities
 
-## :gem:Job boards
+---
 
-| Geo                               | Name & Link                                                             | Specialty          |
-| --------------                    | ------------------------------------                                    | --------------     |
-|                                   | [Women Who Code](https://www.womenwhocode.com/jobs)                     | Womxn:rainbow:     |
-|                                   | [Diversity in Tech](https://www.diversifytech.co/job-board/entry-level) | Diversity:rainbow: |
-|                                   | [Protege](https://protege.dev/job-board)                                | Diversity:rainbow: |
-|                                   | [No CS OK](https://nocsok.com/)                                         | Self-taught & bootcamp |
-|                                   | [AngelList](https://angel.co/jobs)                                      | Startups           |
-|                                   | [Product Hunt](https://www.producthunt.com/jobs)                        | Startups           |
-| :triangular_flag_on_post::us:     | [Built In](https://builtin.com/tech-hubs)                               | Startups           |
-| :triangular_flag_on_post::canada: | [TechTO Career newsletter](https://www.techto.org/newsletter)           | Startups           |
-| :triangular_flag_on_post::canada: | [Work in Tech](https://www1.communitech.ca/jobs?filter=eyJqb2JfZnVuY3Rpb25zIjpbIlNvZnR3YXJlJTIwRW5naW5lZXJpbmciXX0%3D&q=junior)           | Startups           |
-|                                   | [The Muse](https://www.themuse.com/)                                    |                    |
-|                                   | [4 Day Week](https://4dayweek.io/)                                      | 4-day work week    |
+<!-- prettier-ignore-start -->
 
+#### :point_right: Discord server invites expire in 7 days, so no server invites are provided. The following contain links to starting points. Discover on your own! :arrow_heading_down: If the community _does not_ have a Discord Server it will be marked with a :no_entry:
 
-## :money_with_wings:Negotiating the offer
+<!-- prettier-ignore-end -->
+
+- **100Devs** :dart: Leon Noel, Managing Director of Engineering @ [Resilient Coders](https://resilientcoders.org/) helps you become a software engineer for free: :point_right: [link](https://leonnoel.com/100devs/) :rainbow:
+- **#DearJuniorDev** Follow [Jessica Parker](https://www.linkedin.com/in/devjessparker/) on LinkedIn.
```

Comment:
I love these suggestions! In terms of classification, maybe this one belongs in a separate category, something like `Content` or `Influencers`?

Reply:
Thank you! I moved DearJuniorDev into its own category `Social Content`. that does make more sense despite there being a discord community as well, its primarily through LinkedIn and in the future more social resources can be added.